### PR TITLE
More expressive invoice descriptions

### LINKED ISSUE #58: [CLN] improve invoice memo
https://github.com/ElementsProject/peerswap/blob/9fb06e17f9bc79f4032c0b4aa9b92c97bb1c1eec/clightning/clightning.go#L250

Print 
- swap-in
- type of swap (swap-in/out)
- asset of swap
- type of invoice (fee/preimbursement/swap)

### PR DESCRIPTION
**Pull request by nepet**
This pull request adds more descriptive invoice messages that resolve #58

### CODE REVIEW

File: swap/actions.go [hunk 1]
```diff
@@ -356,20 +357,15 @@ func (c *CreateSwapOutFromRequestAction) Execute(services *SwapServices, swap *S
...
 	}
 
-	/*
-		 feeSat, err := services.policy.GetMakerFee(swap.Amount, swap.OpeningTxFee)
-		 if err != nil {
-
-		return Event_ActionFailed
-		}
-	*/
+	// Parse memo
```
Comment (wtogami) [inline swap/actions.go:360]: Minor nit: "Parse" means interpret, you're doing the opposite, a formatted print

**Comment by wtogami** [comment]
Tested ... this is a vast improvement over erroneous `liquid swap` on all CLN invoices prior.

Unfortunately this is bothering me ...

My `swap-out` is a `swap-in` from the partner's POV.
My `swap-in` is a `swap-out` from the partner's POV.

Invoice labels `swap-out` or `swap-in` are generated from the POV ... of the requester? (confused about this)

Invoice label: `swap-out lbtc fee fbd2c20f5461a567909d19a0dd742db0ad91efbcbddc4ac0b2b88f07517dbfd8`

It bothers me that these invoice labels show `swap-out` or `swap-in` to both sides when that isn't what it is from the POV of one side.

```
      "type": "swap-out",
      "role": "receiver",
```

Within `listswaps` and the statistics for each peer in `listpeers` it seems we get this POV right by differentiating the type and the role?

Since the invoice text is identical to both sides, how about this instead?

Invoice label: `peerswap lbtc fee 699350x125x0 fbd2c20f5461a567909d19a0dd742db0ad91efbcbddc4ac0b2b88f07517dbfd8`
Invoice label: `peerswap lbtc claim 699350x125x0 fbd2c20f5461a567909d19a0dd742db0ad91efbcbddc4ac0b2b88f07517dbfd8`

It was peerwap ... the short-id is often more useful at a glance ... and they can lookup the swap id if they want to.

I think after unifying the types in the invoice labels this is ready for merge.

**Comment by wtogami** [comment]
After this is merged please rebase #83 and #97

**Comment by nepet** [comment]
Requested changes are done. Self Ack 6e22105723617df2fff6515c4087863094789998

**Pull request merged**